roles/ansible_service_broker/defaults/main.yml

```diff
@@ -22,6 +22,9 @@ ansible_service_broker_sandbox_role: edit
 ansible_service_broker_auto_escalate: false
 ansible_service_broker_local_registry_whitelist: []
 
+
+ansible_service_broker_full_broker_config_map: "{{ lookup('template', 'configmap.yaml.j2') | from_yaml }}"
```

Comment:
I would remove this variable here and add it as a var on the configmap creation task, because some of the variables used in the configmap are defined as the role runs.